src/video/sdl2_v.cpp

```diff
@@ -560,12 +576,10 @@ std::optional<std::string_view> VideoDriver_SDL_Base::Start(const StringList &pa
 	if (error) return error;
 
 #ifdef SDL_HINT_MOUSE_AUTO_CAPTURE
-	if (GetDriverParamBool(param, "no_mouse_capture")) {
```

Comment:
Is removing this parameter intended? There is nothing about it in the PR description.